src/js/components/Diagram/doc.js

```diff
@@ -18,6 +18,16 @@ export const doc = Diagram => {
     connections: PropTypes.arrayOf(
       PropTypes.shape({
         anchor: PropTypes.oneOf(['center', 'vertical', 'horizontal']),
+        animation: PropTypes.oneOfType([
```

Comment:
In addition to this definition at the connection level, I was expecting documentation for `animation` to also be shown as a sibling to `connections`.

Reply:
I would agree here. It may be helpful to convert your Progressing.js story into a .tsx to make sure the type definitions are all accounted for